### DISCUSSION TITLE
Allow access to presenter window

### DISCUSSION THREAD

**Discussion by lulalala** [marp-team/marp#441] [Ideas] [answered]
I have some script to play animation in my slides, and I wish that some custom events can be passed from main window to the presenter window. Could the presenter window be made into a globally accessible variable?

**Comment by yhatt** [comment, marked as answer]
No. We have not designed the presenter window provided by Marp CLI's `bespoke` template, to allow customization by users.

Marp ecosystem is basically only have responsible only for rendering slides, and we are assuming the output of Marp (Core) to integrate with UI such as the presenter window brought from others.

If you want advanced synchronization with the presenter window, please consider to create a Marp integrated app with the library that provides the presenter window (Reveal.js, Spectacle, Slidev, etc).
